src/serializers/styleSheetSerializer.js

```diff
@@ -38,8 +38,39 @@ const getMediaQueries = (ast, filter) => (
     }, [])
 )
 
-const getStyles = (classNames) => {
-  const styles = styleSheet.rules().map(rule => rule.cssText).join('\n')
+const resolveDeclarations = declarations => {
+  const declarationsMap = {}
+  return declarations
+    .reverse()
+    .filter(declaration => {
+      if (declarationsMap[declaration.property]) {
+        return false
+      }
+
+      declarationsMap[declaration.property] = true
+      return true
+    })
+    .reverse()
```

Comment:
The two reverses are a tad novel in this function and could be done with a simple reverse loop. Sometimes I like novelty :D